### PR TITLE
feat(components): select bar

### PR DESCRIPTION
**Pull request by dahaca**
## Description

* refresh styles
* add puck movement animation
* add keyboard controls

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/9003

## Screenshots:

* see storybook

### CODE REVIEW

File: packages/components/src/components/form/SelectBar/SelectBar.stories.tsx [hunk 1]
```diff
@@ -11,30 +12,21 @@ const options = [
...
-            <SelectBarComponent
-                label={args.label}
-                selectedOption={option}
-                options={options}
-                onChange={setOption}
-            />
-        );
+        return <SelectBarComponent {...args} onChange={setOption} options={options} />;
+    },
+    argTypes: {
```
Comment (AdamSchinzel) [inline packages/components/src/components/form/SelectBar/SelectBar.stories.tsx:27]:
```suggestion
    argTypes: {
        label: {
            type: 'string',
        },
```

File: packages/components/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
         "@trezor/dom-utils": "workspace:*",
         "@trezor/env-utils": "workspace:*",
         "@trezor/react-utils": "workspace:*",
+        "@trezor/styles": "workspace:^",
```
Comment (AdamSchinzel) [inline packages/components/package.json]: i think it should be `workspace:*`

**Review by AdamSchinzel — CHANGES_REQUESTED** [review]
I see that you wanted to make it look good on smaller devices but unfortunately it breaks on smaller screens.

The old component looked like this:
45945FAE-BBD2-474C-935B-8A07DCA9466D

Now it looks:
177BECED-6B22-4C12-A7E6-CD285CC15921_4_5005_c

File: packages/components/src/components/form/SelectBar/SelectBar.stories.tsx (hunk 1 above)
Comment (dahaca) [reply packages/components/src/components/form/SelectBar/SelectBar.stories.tsx:27]: 9a5dfefcce

Fixed the mobile design as well! It was actually deleted in the keyboard nav commit by accident 😓

File: packages/components/package.json (hunk 2 above)
Comment (dahaca) [reply packages/components/package.json]: 9a5dfefcce

**Review by AdamSchinzel — APPROVED** [review]
lgtm

**Pull request merged**